test-case-reporting/src/test_result_record.ts

```diff
@@ -315,8 +315,15 @@ export class TestResultRecord {
       .where('test_case_stats.sample_size', sampleSize)
       .join('test_cases', 'test_cases.id', 'test_case_stats.test_case_id')
       .select<Array<DB.TestCase>>(
-        this.db.raw('passed + failed + skipped + errored AS total'),
-        this.db.raw(`${stat} / total AS ${stat}_percent`)
+        this.db.raw('?? / (?? + ?? + ?? + ??) AS ??', [
+          stat,
+          'passed',
+          'failed',
+          'skipped',
+          'errored',
+          `${stat}_percent`,
+        ]),
+        this.db.raw('*')
```

Comment:
is `raw` necessary for the `*`?

Reply:
I don't know; I made it raw for parity with the other part of the select query. Want me to change it?

Reply:
Adding a function call that does nothing and is not necessary is no bueno :upside_down_face:

Reply:
You should know if it's necessary, and if it works without

Reply:
Done.